app/javascript/packs/OAS3Autocomplete.js

```diff
@@ -59,8 +59,11 @@ const injectParametersToPath = (path: PathItemObject, commonParameters?: Array<P
   }, {})
 )
 
-const injectAutocompleteToResponseBody = (responseBody: ResponseBody, accountData: AccountData): ResponseBody => (
-  {
+const injectAutocompleteToResponseBody = (responseBody: ResponseBody | string, accountData: AccountData): ResponseBody | string => {
+  if (typeof responseBody === 'string') {
+    return responseBody
+  }
+  const res = (responseBody.paths && accountData) ? {
```

Comment:
```suggestion
  const res = (typeof responseBody !== 'string' && responseBody.paths && accountData) ? {
```
Alternatively add it to guard in line 63
```
if (typeof responseBody === 'string' || !responseBody.paths || !accountData) {
  return responseBody
}
```